src/scenic/core/regions.py

```diff
@@ -1185,6 +1189,20 @@ def intersects(self, other, triedReversed=False):
                 return surface_collision
 
             # PASS 4
+            # If we have 2 candidate points and both regions have only one body,
+            # we can just check if either region contains the candidate point of the
+            # other.
```

Comment:
To make the reasoning clearer I'd add something like "(since surface intersections were ruled out above, so if the regions intersect then one contains the other)".